react-common/components/controls/Modal.tsx

```diff
@@ -127,5 +127,5 @@ export const Modal = (props: ModalProps) => {
                 </div>
             }
         </div>
-    </FocusTrap>, parentElement || document.body)
+    </FocusTrap>, parentElement || document.getElementById("root") || document.body)
```

Comment:
to clarify a bit in case anyone's confused what this actually changes; `parentElement` is an optional prop that determines where this this modal should be rooted (as modals are things that should 'show up' above other things and so they need to get positioned at a different place in the dom). When we were looking at it yesterday we noticed that not all modals were passing a value for this and were instead falling back to being placed under the body instead of root where `.hc` gets applied (`#root` is the `ProjectView`'s id in `app.tsx`).

If we're in the webapp almost everything is intended to be under root anyway which is why I suggested this vs going back and making `parentElement` required~